themes/doks/layouts/_default/_markup/render-heading.html

```diff
@@ -29,4 +30,15 @@
       <img src="/icons/chain-icon.png" class="chain-icon" alt="heading-link-icon" title="Click to copy the section link." />
     </a>
   </h{{ .Level }}>
+  {{ end }}
+{{ else }}
+  {{ $anchor := strings.ReplaceRE `-hahahugoshortcode-s.-hbhb` "" .Anchor}}
+  <h{{ .Level }} id="{{ $anchor | safeURL }}"  class="link-heading">
+    <a href="#{{ $anchor | safeURL }}">
+      {{ .Text | safeHTML }} 
+    </a>
+    <a href="#{{ $anchor | safeURL }}" class="chain-icon-link" onclick="copyLink(this)">
+      <img src="/icons/chain-icon.png" class="chain-icon" alt="heading-link-icon" title="Click to copy the section link." />
+    </a>
+  </h{{ .Level }}>
 {{ end }}
```

Comment:
R U sure you need to use {{end}} and then {{else}} and not {{else}} straight away ?
Also - please indent the else block to align with the if block

Reply:
There's a nested if.. maybe it's clearer now?